File: EmployeeManagement/Repositories/PositionRepository.cs
using System.Data.Entity;
using System.Linq;
using EmployeeManagement.Models;
using EmployeeManagement.Repositories.Interfaces;

namespace EmployeeManagement.Repositories
{
    public class PositionRepository : Repository<Position>, IPositionRepository
    {
        public bool Exists(int id)
        {
            return DbSet.Count(e => e.Id == id) > 0;
        }        
    }
}

File: EmployeeManagement/App/repositories/PositionRepository.ts
import { Repository } from './repository';
import { Position as PositionModel } from '../models/Position';
import { IRawPosition } from '../models/raw/IRawPosition';
import { IPositionRepository } from './interfaces/IPositionRepository';

export class PositionRepository extends Repository<IRawPosition, PositionModel> implements IPositionRepository {
    protected specificUrl(): string {
        return 'position';
    }
}